release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -76,6 +76,12 @@ All {product-title} 4.11 clusters require this administrator acknowledgment befo
 
 For more information, see xref:../updating/updating-cluster-prepare.adoc#updating-cluster-prepare[Preparing to update to {product-title} 4.12].
 
+[id="ocp-4-12-feature-set"]
+==== Enabling a feature set when installing a cluster
+Beginning with {product-title} {product-version}, you can now enable a feature set as part of the installation process. A feature set is a collection of {product-title} features that are not enabled by default.
+
+For more information about enabling a feature set during installation, see xref:../nodes/clusters/nodes-cluster-enabling-features.adoc#nodes-cluster-enabling[Enabling features using feature gates].
```

Comment:
The actual name of the linked section is "Enabling OpenShift Container Platform features using FeatureGates", so I'd recommend using that exactly. (Likely with {product-title} though of course)